src/test/java/com/scalefocus/libraryproject/BasicTests.java

```diff
@@ -0,0 +1,48 @@
+package com.scalefocus.libraryproject;
```

Comment:
please, move this test suite to an according package. you can mirror the production code packages. E.g. you can create a package com.scalefocus.libraryproject.services under src/main/test directory.